sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/WebPubSubClient.cs

```diff
@@ -28,24 +28,60 @@ namespace Azure.Messaging.WebPubSub.Clients
     public class WebPubSubClient
     {
         private const string ErrorNameDuplicate = "Duplicate";
+        private const string RecoverConnectionIdQuery = "awps_connection_id";
+        private const string RecoverReconnectionTokenQuery = "awps_reconnection_token";
 
         // Some exposed properties for testing
         internal IWebSocketClientFactory WebSocketClientFactory { get; set; }
         internal TimeSpan RecoverDelay { get; set; } = TimeSpan.FromSeconds(1);
+        internal ulong CurrentSequenceId => _sequenceId.Current;
+
+        private static readonly UnboundedChannelOptions s_unboundedChannelOptions = new UnboundedChannelOptions
```

Comment:
i dont think we need cache it. each client only creates two option instances.

Reply:
I don't mean to cache it, but it makes codes look clean